mypyc/primitives/registry.py

```diff
@@ -367,7 +369,7 @@ def c_method_op(name: str,
         priority: if multiple ops match, the one with the highest priority is picked
```

Comment:
Document `truncated_type`. Mention that if defined, it and return type must be non-reference counted integer types (or bool).